sci-astronomy/fitspng/fitspng-1.4.recipe

```diff
@@ -0,0 +1,58 @@
+SUMMARY="Utility to convert images in astronomical FITS to PNG format"
+DESCRIPTION="FITS format is a general purpose astronomical image format.
+FITS images stores measurements of a physical quantity - an array of the
+angular distribution of the photon events or the optical intensity.
+Fitspng does visualization of such array by converting them to the
+images in PNG format."
+HOMEPAGE="http://integral.physics.muni.cz/fitspng/"
+COPYRIGHT="2006 - 2019, F. Hroch,
+	Institute of Theoretical Physics and Astrophysics,
+	Masaryk University, Brno, Czech Republic."
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="ftp://integral.physics.muni.cz/pub/fitspng/fitspng-$portVersion.tar.gz"
+CHECKSUM_SHA256="caa7fc2f0c90bbd14b9499a705d74bb07df9f6ecf53454b9614ec3b30459f89e"
+ARCHITECTURES="!x86_gcc2 x86_64 ?arm ?ppc ?sparc"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	fitspng${secondaryArchSuffix} = $portVersion
+	cmd:fitspng${secondaryArchSuffix} = $portVersion
+	"
+REQUIRES="
+	haiku${secondaryArchSuffix}
+	lib:libcfitsio${secondaryArchSuffix} >= 3.47
+	lib:libpng${secondaryArchSuffix}
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libcfitsio${secondaryArchSuffix} >= 3.47
+	devel:libpng${secondaryArchSuffix}
+	"
+BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:autoreconf
+	cmd:awk
+	cmd:gcc${secondaryArchSuffix}
+	cmd:make
+	"
+
+BUILD()
+{
+	if [ $effectiveTargetArchitecture == x86_64 ]; then
+		GCC5FLAGS="--enable-x86-64"
+	fi
+
+	CFLAGS="-O4 -DNEBUG"
+
+	autoreconf -vfi
+	runConfigure ./configure
+
+	make $jobArgs
+}
+
+INSTALL()
+{
+	make install
```

Comment:
In the first draft you made you also installed the html file to $docDir, maybe also do this now? (also the images for it)
Probably need to go into $docDir/fitspng/* (haven't checked there yet)

Reply:
Strike this :) **make install** takes care of installing the documentation and the manpage :)